### PR TITLE
Revise SFTP Warning and Recommendations

### LINKED ISSUE #769: Document Recommended Connection Settings for SFTP Client
Document recommended settings for SFTP clients: advanced  default remote directory is blank and limit the number of simultaneous connections to 1 

Add to existing sftp connection doc where appropriate. make sure warning exists about migrating app servers and credentials changing periodically.

Revise [SFTP FAQs](https://pantheon.io/docs/articles/sites/sftp-faqs#system-maintenance) - no questions. consider changing doc slug title
Is this still a thing? 
>We're performing system maintenance on your development environment right now which prevents us from being able to commit your changes. The system maintenance shouldn't take long and this message will disappear once it's done.

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #769 
- Delete: [SFTP FAQs](https://pantheon.io/docs/articles/sites/sftp-faqs)

For [Developing On Pantheon Directly with SFTP Mode](https://pantheon.io/docs/articles/sites/code/developing-directly-with-sftp-mode#troubleshooting):
- Add Subsection: **Connection Issues** (Preserved from SFTP FAQs)
- Revise Warning - migrated app servers do not nuke uncommitted changes, they are kept. Changes are not included in backups and will be deleted after confirmation when changing connection mode to Git. 

@atjuch @newtoid can you review and +1? 

**Redirects Needed:**
No relative links found for `/docs/articles/sites/sftp-faqs` - add redirect to production site once deployed 
- [SFTP FAQs](https://pantheon.io/docs/articles/sites/sftp-faqs) to [Developing On Pantheon Directly with SFTP Mode](https://pantheon.io/docs/articles/sites/code/developing-directly-with-sftp-mode)

### CODE REVIEW

**Comment by newtoid** [comment]
+1 huge emphasis on good practice 'You should commit early and commit often.' in the event of calamity everything in version control is safe, sftp changes even if preserved post migrate can't be rolled back

File: source/docs/articles/sites/code/developing-directly-with-sftp-mode.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Due to the nature of our platform, the connection information will change from t
 
  ![Enable SFTP mode](/source/docs/assets/images/desk_images/278855.png)
 <div class="alert alert-danger" role="alert"><h4>Warning</h4>
-SFTP changes to code that have not been committed <em>will not be saved in backups</em> as they are not part of your code repository yet. If you have pending changes and the site is migrated (which we may do at any time for various reasons), your changes will be lost without warning. You must commit these file changes if you want to keep them permanently. If you switch from SFTP to Git mode while uncommitted changes are pending, you will be presented with a dialog confirming that you don't want to keep these changes, and if confirmed, they will be deleted.</div>
```
Comment (bmackinney) [inline source/docs/articles/sites/code/developing-directly-with-sftp-mode.md]: I'd like to get more feedback on whether this is no longer true. @newtoid I saw you commented in Slack earlier. Do you remember where you heard that from?

**Comment by atjuch** [comment]
+1

File: source/docs/articles/sites/code/developing-directly-with-sftp-mode.md (hunk 1 above)
Comment (bmackinney) [inline source/docs/articles/sites/code/developing-directly-with-sftp-mode.md]: It's internally documented. Thanks @newtoid

**Comment by bmackinney** [comment]
:100:

**Pull request merged**